app/helpers/contacts_helper.rb

```diff
@@ -9,4 +9,11 @@ def start_a_conversation_link(aspect, contacts_size)
                   title: t("contacts.index.start_a_conversation")
     end
   end
+
+  def gon_load_contact(contact)
+    Gon.preloads[:contacts_collection] ||= []
+    if contact.id.nil? || !Gon.preloads[:contacts_collection].map {|contact| contact[:id] }.include?(contact.id)
```

Comment:
So you want to check for a `Contact.new`, that's what [`persisted?`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-persisted-3F) is for